services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/functionaltests/BasicCrudTest.java

```diff
@@ -222,6 +220,25 @@ public void deleteTable() {
                                                           .build());
     }
 
+    @Test
+    public void getItemInterfaceToOperationSucceeds() {
+        mappedTable.getItem(r -> r.key(Key.create(stringValue("id-value"), stringValue("sort-value"))));
+        mappedTable.getItem(GetItemEnhancedRequest.builder()
+                                                  .key(Key.create(stringValue("id-value"), stringValue("sort-value")))
+                                                  .build());
+    }
+
+    @Test
+    public void putItemInterfaceToOperationSucceeds() {
+        Record record = new Record()
+            .setId("id-value")
+            .setSort("sort-value")
+            .setAttribute("one");
+
+        mappedTable.putItem(Record.class, r -> r.item(record));
+        mappedTable.putItem(PutItemEnhancedRequest.builder(Record.class).item(record).build());
+    }
+
```

Comment:
I think we should be able to make basic assertions here beyond 'it didn't blow up'.